contrib/build_sysimg.jl

```diff
@@ -71,18 +71,18 @@ function build_sysimg(sysimg_path=nothing, cpu_target="native", userimg_path=not
         try
             # Start by building inference0.{ji,o}
             inference0_path = joinpath(dirname(sysimg_path), "inference0")
-            info("Building inference0.o...")
+            info("Building inference0.o")
             println("$julia -C $cpu_target --output-ji $inference0_path.ji --output-o $inference0_path.o coreimg.jl")
             run(`$julia -C $cpu_target --output-ji $inference0_path.ji --output-o $inference0_path.o coreimg.jl`)
```

Comment:
Yeah, i'll remove it. The only reference I have explaining inference0 is here https://github.com/JuliaLang/julia/blob/6b5a05eb1a029aef93f77b60bb1d745c7d6e1d8d/doc/devdocs/functions.rst 
(scroll all the way down)
It doesn't show up in any other search.